packages/portfolio-contract/src/portfolio.contract.ts

```diff
@@ -72,14 +92,16 @@ export const contract = async (
     makeOpenPortfolioInvitation() {
       trace('makeOpenPortfolioInvitation');
       return zcf.makeInvitation(
-        (seat, offerArgs) =>
-          openPortfolio(
+        (seat, offerArgs: OfferArgsShapes) => {
+          mustMatch(offerArgs, offerArgsShapes);
+          return openPortfolio(
             seat,
             offerArgs,
             localV as unknown as Promise<
               OrchestrationAccount<{ chainId: 'agoric-any' }>
```

Comment:
`OrchestrationAccount<{ chainId: 'agoric-any' }>` is a mouthful. I made a `LocalAccount` alias in my suggestion for the exo. I wonder about using it more widely.